toolbox/siesta/barriers/BarriersBase.py

```diff
@@ -52,7 +51,7 @@ class SiestaBarriersBase():
     ghost_info                   :
     """
     def __init__(self,
-                 host_path = None,
+                 host_path = "./",
```

Comment:
If you use `pathlib.Path` it would be best to simply do: `host_path=''` since `pathlib.Path('')` is the current directory (by default).